resources/publications.md

```diff
@@ -95,12 +93,35 @@ td:nth-child(2) {
 
 ## Metrics
 
+### Learnable metrics
+
+Directly optimize correlation with human judgments.
+
 |     |     |     |
 | --- | --- | --- |
 | 2020 | [***COMET: A Neural Framework for MT Evaluation***](https://aclanthology.org/2020.emnlp-main.213.pdf) | Ricardo Rei, Craig Stewart, Ana C. Farinha, Alon Lavie |
+| 2020 | [***BLEURT: Learning Robust Metrics for Text Generation***](https://aclanthology.org/2020.acl-main.704.pdf) | Thibault Sellam, Dipanjan Das, and Ankur Parikh |
+
+### Embedding-based metrics
+
+Use various word embeddings as an alternative to n-gram matching for capturing word semantics similarity.
+
+|     |     |     |
+| --- | --- | --- |
+| 2020 | [***Bertscore: Evaluating text generation with bert***](https://openreview.net/forum?id=SkeHuCVFDr) | Tianyi Zhang, Varsha Kishore, Felix Wu, Kilian Q. Weinberger, and Yoav Artzi |
+| 2019 | [***MoverScore: Text generation evaluating with contextualized embeddings and earth mover distance***](https://aclanthology.org/D19-1053.pdf) | Wei Zhao, Maxime Peyrard, Fei Liu, Yang Gao, Christian M. Meyer, Steffen Eger |
+
+
+### n-gram matching metrics
+
+Using hand-crafted features, estimate MT quality by counting the number and fraction of n-grams that appear simultaneous in a candidate translation hypothesis and one or more human-generated references.
```

Comment:
> machine translation

We don't use "MT".  Also it's a bit odd that in the other descriptions, machine translation was not mentioned, just implied.  So maybe it can be here too?  e.g.

> n-gram matching metrics use hand-crafted features to estimate quality.

> They require a human reference translation.

> For example, a metric can count the number of words that appear in both the machine translation and the human translation.

> simultaneously

But I would love to make this sentence shorter and simpler

Reply:
It's good but I am paranoid about potentially ambiguous parses ;-)

So we avoid "как в машине, так и в человеческом переводе".

Probably fine (I tested this one, it didn't happen).

Reply:
Just in case, will change to "both the machine translation and the human translation."